### PR TITLE
Added wmutils

### PR DESCRIPTION
**Pull request by periish**
Read: https://k1ss.org/style
Read: https://k1ss.org/guidestones

---

## Description of package
Shell commands to control X windows.

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/wmutils-core/build [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+make
+make prefix="/usr" DESTDIR="$1" install
```
Comment (dylanaraps) [inline community/wmutils-core/build]: Drop quotes around `/usr`.

Comment (dylanaraps) [reply community/wmutils-core/build]: Otherwise LGTM

File: community/wmutils-core/depends [hunk 2]
```diff
@@ -0,0 +1,3 @@
+libxcb
+xcb-util-xrm
+xcb-util-cursor
```
Comment (dylanaraps) [inline community/wmutils-core/depends]: Sort this list.

Comment (periish) [reply community/wmutils-core/depends]: Done.

**Pull request merged**